harvester_e2e_tests/integration/test_hosts.py

```diff
@@ -67,3 +178,150 @@ def test_host_poweron_state(api_client, host_state, wait_timeout):
 
     _, node = api_client.hosts.get(node['id'])
     assert "active" == node["metadata"]["state"]["name"]
+
+
+@pytest.mark.hosts
+@pytest.mark.p0
+def test_verify_host_info(api_client):
+    status_code, nodes_info = api_client.hosts.get()
+
+    assert status_code == 200, f"Failed to list nodes with error: {nodes_info}"
+    assert len(nodes_info['data']) >= 1, "Incorrect hosts amount"
+
+    fails = []
+    for node in nodes_info['data']:
+        fail = []
+        node_name = node.get('metadata', {}).get('name', "")
+        if node_name == "":
+            fail.append("Node name can't be empty")
+
+        cpus = node.get('status', {}).get('capacity', {}).get('cpu', 0)
+        if cpus == 0:
+            fail.append("Node {node_name} cpu should not be zero")
+
+        mems = node.get('status', {}).get('capacity', {}).get('memory', "0Ki")
+        mems = "".join(c for c in mems if c.isdigit())
+        mems = mems or "0"
+        if int(mems, 10) == 0:
+            fail.append("Node {node_name} memory should not be zero")
+
+        if fail:
+            fails.append(fail)
+
+    assert not fails, (
+        "Failed to get node information with errors:\n",
+        "\n".join(fails)
```

Comment:
I thought here might raise another exception: `TypeError: sequence item 0: expected str instance, list found`

If node name be included in error message, I think we can just append into `fails`.
And we should also consider the case that `node_name` is empty. (to fall back to use `data['id']` like `f"ID: {data['id']}`"?)